portfolio/src/main/webapp/index.html

```diff
@@ -174,9 +173,39 @@ <h1 class="title">
       <div id="videos" class="videos--list"></div>
     </div>
   </section>
-  <section class="section animate-2 animate--fade-in-left">
-    <div class="container" id="quote-content"></div>
-  </section>
+  <div class="section animate-2 animate--fade-in-left">
+    <div class="container">
+      <p class="title mb-4 is-size-4">
+        Comments
+      </p>
+      <div id="comments" class="mt-3">
+      </div>
+      <article class="media mt-3">
+        <figure class="media-left">
+          <p class="image is-64x64">
+            <img src="./images/avatar-placeholder.png">
```

Comment:
Add alt text here? Even if it's just a placeholder for now

Reply:
Done!